Libs/Gst/Classes/Parse.cs

```diff
@@ -1,16 +1,16 @@
-using Gst;
+using System;
 
 namespace Gst
 {
     public class Parse
     {
         public static Element Launch(string pipelineDescription)
```

Comment:
We need to handle the case where `ret` is NULL, and return appropriately. Should probably be `Element?`.

Reply:
As `Global.parse_launch` returns an error pointer I'm now throwing an Exception in case of an error. Otherwise we would lose the error information.